packages/lesswrong/server/repos/UsersRepo.ts

```diff
@@ -284,6 +284,31 @@ export default class UsersRepo extends AbstractRepo<DbUser> {
     `)
   }  
 
+  async getUsersWithNewDialogueChecks(minutes: number): Promise<DbUser[]> {
+    return this.manyOrNone(`
+      SELECT "Users".*
+      FROM "Users"
+      INNER JOIN "DialogueChecks" ON "Users"._id = "DialogueChecks"."targetUserId"
+      WHERE
+          "DialogueChecks"."checkedAt" > NOW() - INTERVAL '$1 minutes'
+          AND "DialogueChecks".checked = true
```

Comment:
always prefer `IS TRUE`; regular equality comparisons for booleans fall over on nulls